package.json

```diff
@@ -53,6 +53,7 @@
     "@woocommerce/product-editor": "^1.4.0",
     "@woocommerce/tracks": "^1.3.0",
     "@wordpress/data": "^7.6.0",
+		"@wordpress/i18n": "^5.6.0",
```

Comment:
The spacing is weird here, in the editor it appears ok. It uses spaces just like the rest of the lines 🤷